opsdroid/connector/matrix/connector.py

```diff
@@ -133,8 +140,23 @@ async def make_filter(self, api, fjson):
 
     async def connect(self):
         """Create connection object with chat library."""
-        config = nio.AsyncClientConfig(encryption_enabled=False, pickle_key="")
-        mapi = nio.AsyncClient(self.homeserver, self.mxid, config=config)
+
+        if not Path(self.store_path).is_dir():
+            Path(self.store_path).mkdir()
+
+        config = nio.AsyncClientConfig(
+            encryption_enabled=True,
+            pickle_key="",
+            store=nio.store.SqliteStore,
+            store_name="test_store",
```

Comment:
```suggestion
            store_name="opsdroid",
```
why not name this something like this?